pages/windows/psping.md

```diff
@@ -0,0 +1,24 @@
+# psping
+
+> A ping tool that includes TCP ping, latency and bandwidth measurement.
+> More information: <https://docs.microsoft.com/en-us/sysinternals/downloads/psping>.
```

Comment:
We've recently started removing the `en-us` locale from the doc links as the Microsoft website will redirect to the user's locale when clicking it. 👍🏻
```suggestion
> More information: <https://docs.microsoft.com/sysinternals/downloads/psping>.
```

Reply:
Ah, good call. Have addressed